### PR TITLE
feat: show timeline only on global context

### LINKED ISSUE #911: The timeline should only be available in a `global` context
Currently, we just support timeline for a global map, so if the path is in a context like: planter, org, wallet, then we should hide the timeline to avoid confusion. like this path: http://localhost:3000/organizations/178/trees/762681

### PR DESCRIPTION
**Pull request by RubenSmn**
# Description

Fixes #911

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by RubenSmn** [comment]
I will fix the checks in a bit

**Comment by dadiorchen** [comment]
@RubenSmn is this pr ready for merge?

**Comment by RubenSmn** [comment]
@dadiorchen yes it only shows on the `top` page now

**Comment by dadiorchen** [comment]
@RubenSmn it should be alse visible on home/index page

**Comment by RubenSmn** [comment]
I will fix that!

File: src/components/Timeline.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ function Timeline() {
     end: Date.now(),
   });
   const { map } = useMapContext();
-  const isGlobalPage = !!router.asPath.match(/\/top/g);
+  const isGlobalPage = !!router.pathname.match(/^(?:\/top|\/)$/g);
```
Comment (dadiorchen) [inline src/components/Timeline.js:71]: @RubenSmn we need to do some extra work on these cases, the path on other envs will be a bit different with local, the root of next.js will be: /web-map-beta/beta as the base path, os we will fail on your code above on other envs, please check my code here to handle it: https://github.com/Greenstand/treetracker-web-map-client/blob/752f85020c90473c4fd56940edb6f6ef82ca58b6/src/components/App.js#L76-L90

Comment (RubenSmn) [reply src/components/Timeline.js]: Thanks for the extra info!

**Comment by dadiorchen** [comment]
@RubenSmn sorry for the delay, can you take a look at the conflict?

**Comment by RubenSmn** [comment]
I will!

**Comment by RubenSmn** [comment]
@dadiorchen fixed the conflicts

**Pull request merged**